### PR TITLE
Translate English version into Japanese

### PR DESCRIPTION
**Pull request by ryoon**
Do not translate the sentenses in code blocks to explain
how to write the commit messages in English.

### CODE REVIEW

**Comment by RomuloOliveira** [comment]
Hi, @ryoon, thanks for your translation! I'm not able to do a review on this PR, do you know any colleague or friend that could help? If you don't, just let me know and I'll merge it.

File: README_ja-JP.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+
+リベースやスカッシュは、公開されたコミットまたは複数の人が作業している共有されたブランチでは避けてください。
+リベースやスカッシュは履歴を書き換え、既存のコミットを上書きします。
+これを共有されたブランチで行うと混乱をまねき(つまり、コミットはリモートリポジトリーにプッシュされるし、リモートリポジトリーは他の人のブランチから生成される)、ツリーの状態が一貫性を失い競合が発生することで、他の人が自分の変更を(ローカルとリモートの両方で)失うかもしれません。
+
+## 有用なgitコマンド
+
+### rebase -i
+
+コミットをスカッシュしたり、メッセージを編集したり、コミットを書き換えたり、削除したり、並べ替えるなどをするのに使います。
```
Comment (nogut0123) [inline README_ja-JP.md]:
```suggestion
コミットをスカッシュしたり、メッセージを編集したり、コミットを書き換えたり、削除したり、並べ替えたりするために使います。
```

File: README_ja-JP.md [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+# If you remove a line here THAT COMMIT WILL BE LOST.
+#
+# However, if you remove everything, the rebase will be aborted.
+#
+# Note that empty commits are commented out
+```
+
+#### fixup
+
+コミットを簡単に、もっと複雑なリベースを必要とせずに、クリーンナップするのを使います。
```
Comment (nogut0123) [inline README_ja-JP.md]:
```suggestion
もっと複雑なリベースを必要とせず、簡単にコミットを綺麗にするために使います。
```

**Comment by nogut0123** [comment]
Nice to meet you @ryoon. I read the translation and send some suggestions. Please approve the changes if there are no problems.

**Comment by ryoon** [comment]
Hi, @RomuloOliveira and @nogut0123,
And thanks for your review, @nogut0123 .

I think that Japanese translation is ready to merge.

Thank you.

**Review by RomuloOliveira — APPROVED** [review]
Thanks @ryoon and @nogut0123

**Pull request merged**